lib/features/onboarding/ui/pages/basic_info_page.dart

```diff
@@ -51,6 +59,7 @@ class _BasicInfoPageState extends State<BasicInfoPage> {
                   subTitle: OnboardingConstants.basicInfoSubtitle,
                 ),
                 _buildInfoContentCard(),
+                SizedBox(height: theme.sizing.height.s10),
```

Comment:
As this sizebox is used for spacing use spacing extension
```suggestion
                SizedBox(height: theme.spacing.height.s40),
```

Reply:
Changed